src/Rand.php

```diff
@@ -56,7 +54,7 @@ public static function getBytes($length, $strong = false)
         if (true === $strong && false === $checkAlternatives) {
             throw new Exception\RuntimeException(
                 'This PHP environment doesn\'t support secure random number generation. ' .
-                'Please consider installing the OpenSSL and/or Mcrypt extensions'
+                'Please consider installing Mcrypt extension or use PHP 7'
```

Comment:
Rephrase this to "Please consider either installing ext/mcrypt or upgrading to PHP 7".